apps/prairielearn/src/schemas/schemas/infoCourse.json

```diff
@@ -68,11 +68,12 @@
       "items": {
         "description": "A single assessment set description.",
         "type": "object",
+        "additionalProperties": true,
```

Comment:
To minimize the diff and avoid confusing end users that look at the raw schemas or the docs generated from them, I think we should post-process the generated schemas to remove `"additionalProperties": true`.

Reply:
I would much rather post-process the schema -> docs generation than post-process the schemas, unless the diff is a concern. The idea is for the docs to completely replace any need to look at the raw schemas.

Reply:
... and if that's not the case, I would love to update the docs so that it isn't needed. JSON Schema is confusing enough to point anyone other than developers at.

Reply:
Post-processing the schema is probably almost trivial and keeps the diff clean, so I'm gonna do it regardless 😂

Reply:
Done in [1403ce6](https://github.com/PrairieLearn/PrairieLearn/pull/11024/commits/1403ce6bc89cc7a13bc71815e0030ec6a175891b) and [2fdb351](https://github.com/PrairieLearn/PrairieLearn/pull/11024/commits/2fdb3514b36442fc02f036126afc6761aeabbce8). I'm hoping that [2fdb351](https://github.com/PrairieLearn/PrairieLearn/pull/11024/commits/2fdb3514b36442fc02f036126afc6761aeabbce8) won't be needed once zod 4 rolls around.